tests/helpers/linux/test_mm.py

```diff
@@ -134,3 +145,14 @@ def test_environ(self):
             proc_environ = f.read().split(b"\0")[:-1]
         task = find_task(self.prog, os.getpid())
         self.assertEqual(environ(task), proc_environ)
+
+    def test_slab_for_each(self):
+        proc_slabinfo_names = get_proc_slabinfo_names()
+        foreach_slab_names = [s.name.string_() for s in for_each_slab_cache(self.prog)]
+        self.assertEqual(proc_slabinfo_names, foreach_slab_names)
```

Comment:
This relies on `for_each_slab_cache()` returning names in the same order as `/proc/slabinfo`. This happens to be true currently since they both iterate through the `slab_caches` list, but I don't think we want to guarantee that. The easy fix is:

```suggestion
        self.assertEqual(sorted(proc_slabinfo_names), sorted(foreach_slab_names))
```

Reply:
Scratch that, apparently you can just do:

```suggestion
        self.assertCountEqual(proc_slabinfo_names, foreach_slab_names)
```